embeddings/get_started_with_embedding_tuning.ipynb

```diff
@@ -0,0 +1,1434 @@
+{
```

Comment:
<p>It could be easier to just use Gemini with a PDF input instead of using Document AI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>